golang/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM dockerfile/ubuntu
+
+MAINTAINER Filip Gospodinov "filip@monetas.net"
+# use bash for docker RUN commands; we want associative arrays
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+# use custom GOPATH such that it does not conlfict with GOPATHs that
+# live inside the host's $HOME which may be bind mounted into docker
+ENV GOPATH $HOME/go_monetas
+ENV GOROOT $HOME/go_dist/go
```

Comment:
/opt possible could work as it's owned by root and docker volumes are mount from the root user.
however the concept docker is pushing is data-only containers 
https://docs.docker.com/userguide/dockervolumes/
we might need more experimenting to figure this out